recipes-omnect/omnect-device-service/omnect-device-service.inc

```diff
@@ -67,6 +67,14 @@ do_install:append:omnect_uboot() {
     install -m 0640 -D ${S}/sudo/omnect-device-service-uboot ${D}${sysconfdir}/sudoers.d/omnect-device-service-bootloader-env
 }
 
+# for post process handling we deploy PV
```

Comment:
out of curiosity: where does this post processing happen?
maybe it is worth mentioning here, too?

Reply:
not in meta-omnect. it is for post processing which doesn't affect the content of the image.
should i remove or rephrase the comment? if rephrase, to what?

Reply:
for an omnect developer like me who doesn't know what that post processing exactly is, a comment would be helpful indeed, but as it's a public repository all other audience doesn't need to (and maybe wouldn't) understand what is meant by it.
so, it's ok for me to leave it as is.